README.md

```diff
@@ -33,7 +33,7 @@ Run [acceptance tests](https://www.terraform.io/docs/extend/testing/acceptance-t
 # In one terminal, run a Grafana container.
 # You may optionally override the image tag...
 # GRAFANA_VERSION=7.1.1 \
-make test-serve
+make test-serv
```

Comment:
fix readme to match the task name https://github.com/grafana/terraform-provider-grafana/blob/master/GNUmakefile#L18